### ISSUE TITLE
[6.3] Infinite reloading (Crash when Twitch can't be loaded)

### ISSUE THREAD

**Issue by Rad586** [TheComputerizer/Music-Triggers#142]
**Describe the bug**
Fabric 1.19.2. It causes infinite texture reloading. 

**To Reproduce**
1. Install:
fabric-api-0.76.1+1.19.2, 
musictriggers-1.19.2-fabric-6.3
theimpossiblelibrary-1.19.2-fabric-0.3.1
2. Launch the game. 

**Additional context**
Log: 
With no Internet connection: 
https://gist.github.com/Rad233/7f127fefee9cbb11b403358adfca2a5e
With Internet connection: 
https://gist.github.com/Rad233/ca2583d22a5ce1f6d80961fa0a8be07e

**Comment by TheComputerizer** [comment]
Twitch seems to be causing issues so I think Ill do a small bugfix update before working on 6.4 to address that and some other stuff

**Comment by TwinIndifferent** [comment]
Doubling for issue. Apparently Twitch integration causes resourcepack reloading and CTD for me when I'm offline, if I try to bring up music config menu. _Literallyunplayable:tm:_

```
-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, Stay_True_1.19.zip, xali's+Enchanted+Books+v0.11.zip, cullleaves/smartleaves, FreshAnimations_v1.8.zip, Rain World Music.zip, minecraft/supporteatinganimation
	Recovery: Yes
	Recovery reason: com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Loading Twitch main page failed.
	at com.sedmelluq.discord.lavaplayer.source.twitch.TwitchStreamAudioSourceManager.initRequestHeaders(TwitchStreamAudioSourceManager.java:205)
	at com.sedmelluq.discord.lavaplayer.source.twitch.TwitchStreamAudioSourceManager.<init>(TwitchStreamAudioSourceManager.java:58)
	at com.sedmelluq.discord.lavaplayer.source.AudioSourceManagers.registerRemoteSources(AudioSourceManagers.java:38)
	at com.sedmelluq.discord.lavaplayer.source.AudioSourceManagers.registerRemoteSources(AudioSourceManagers.java:23)
	at mods.thecomputerizer.musictriggers.client.channels.JukeboxChannel.<init>(JukeboxChannel.java:31)
	at mods.thecomputerizer.musictriggers.client.channels.ChannelManager.preInit(ChannelManager.java:95)
	at mods.thecomputerizer.musictriggers.client.channels.ChannelManager.initClient(ChannelManager.java:108)
	at mods.thecomputerizer.musictriggers.MusicTriggersClient.onResourcesLoaded(MusicTriggersClient.java:56)
	at mods.thecomputerizer.theimpossiblelibrary.events.ResourcesLoadedEvent.lambda$static$0(ResourcesLoadedEvent.java:10)
	at net.minecraft.class_6360.handler$fmn000$theimpossiblelibrary$finishReload(class_6360.java:532)
	at net.minecraft.class_6360.method_36562(class_6360.java:46)
	at net.minecraft.class_310.method_29338(class_310.java:654)
	at net.minecraft.class_156.method_17974(class_156.java:541)
	at net.minecraft.class_310.method_24040(class_310.java:650)
	at net.minecraft.class_425.method_25394(class_425.java:146)
	at net.minecraft.class_757.method_3192(class_757.java:870)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.4.2-original
```

**Comment by TheComputerizer** [comment]
Fixed in 6.3.1
